### PR TITLE
Fix segmentation fault in Date helper functions

### PR DESCRIPTION
**Pull request by szledan**
JerryScript-DCO-1.0-Signed-off-by: Szilard Ledan szledan.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by szledan** [comment]
Fix for #413

File: tests/jerry/date-setters.js [hunk 1]
```diff
@@ -208,3 +208,20 @@ assert (d.getUTCDate() == 31);
 d.setTime(0);
 assert (d.setUTCFullYear(1970) == 0);
 assert (d.getUTCFullYear() == 1970);
+
+/* Without argument */
```
Comment (galpeter) [inline tests/jerry/date-setters.js]: We should have tests on real date object and not just on the prototype.

**Comment by galpeter** [comment]
looks good to me

**Comment by zherczeg** [comment]
+1 LGTM

**Comment by egavrin** [comment]
Good to me

**Pull request merged**